src/Appwrite/Platform/Modules/Console/Http/Resources/Get.php

```diff
@@ -67,10 +68,11 @@ public function action(
         Database $dbForPlatform
     ) {
         if ($type === 'rules') {
-            $validator = new Domain($value);
+            $domainValidator = new Domain($value);
```

Comment:
When you are extending the `Domain` class, then you don't need to use both. Just use `AppwriteDomainValidator`